README.md

```diff
@@ -15,7 +15,7 @@ This extension for VS Code makes it easier to split components in an Angular pro
 
 ## How to use
 
-![Preview](./images/preview.gif)
+![Preview](https://github.com/PKief/vscode-angular-component-extractor/raw/develop/images/preview.gif)
```

Comment:
Why do you directly reference the dev branch? If this is really needed, then shouldn't it be the master branch?

Reply:
Fixed it to use a relative branch again. The image will be bundled into the extension package. For now that's okay and we are getting sure that the image will not be lost if it will be removed on the branch. In addition other extensions on the Marketplace handle it similarly.